src/graphql/account.rs

```diff
@@ -177,12 +177,16 @@ impl AccountMutation {
         theme: Option<String>,
         allow_access_from: Option<Vec<IpAddress>>,
         max_parallel_sessions: Option<u8>,
+        customer_ids: Option<Vec<u32>>,
     ) -> Result<String> {
         let store = crate::graphql::get_store(ctx).await?;
         let table = store.account_map();
         if table.contains(&username)? {
             return Err("account already exists".into());
         }
+        if customer_ids.is_none() && role != Role::SystemAdministrator {
+            return Err("You must provide at least one customer id to insert an account that is not in the System Administrator role.".into());
```

Comment:
@kimhanbeom @sophie-cluml 

We don't allow anyone other than system administrators to access all customers setting `customers_ids` to `None`. However, we don't say that users must provide at least one customer, because review-web allows an empty vector of customers. In that case, the user won't be able to access any customers, even though the UI should prevent this. Therefore, the message should be rephrased as:

You are not allowed to access all customers.

Reply:
done